projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -1140,6 +1128,28 @@ public function save_meta( $post_id, $post ) {
 			$submit_post = false;
 		}
 
+		return $submit_post;
+	}
+
+	/**
+	 * Fires when a post is saved, checks conditions and saves state in postmeta so that it
+	 * can be picked up later by @see ::publicize_post() on WordPress.com codebase.
+	 *
+	 * Attached to the `save_post` action.
+	 *
+	 * @param int     $post_id Post ID.
+	 * @param WP_Post $post Post object.
+	 */
+	public function save_meta( $post_id, $post ) {
+		$cron_user   = null;
+		$submit_post = true;
+
+		if ( ! $this->post_type_is_publicizeable( $post->post_type ) ) {
+			return;
+		}
+
+		$submit_post = $this->should_submit_post_pre_checks( $post_id, $post );
```

Comment:
Should this be `$submit_post = $this->should_submit_post_pre_checks( $post);` instead?